src/anemoi/inference/grib/templates/__init__.py

```diff
@@ -54,7 +54,10 @@ def __init__(self, manager: Any) -> None:
         """
         self.manager = manager
 
-    def template(self, variable: str, lookup: dict[str, Any]) -> ekd.Field:
+    def __repr__(self):
+        return f"{self.__class__.__name__}"
```

Comment:
It could be helpful to see the kwargs in the string representation (assuming it will be used in logging somewhere?).